inc/classes/class-srm-post-type.php

```diff
@@ -582,6 +590,12 @@ public function redirect_rule_metabox( $post ) {
 			<em><?php esc_html_e( "If you don't know what this is, leave it as is.", 'safe-redirect-manager' ); ?></em>
 		</p>
 
+		<p id="srm_redirect_rule_message_container">
+			<label for="srm_redirect_rule_message"><strong><?php esc_html_e( 'Message:', 'safe-redirect-manager' ); ?></strong></label>
+			<textarea name="srm_redirect_rule_message" id="srm_redirect_rule_message" class="widefat"><?php echo esc_attr( $redirect_message ); ?></textarea>
```

Comment:
Seems like we may want to use `esc_textarea` here instead of `esc_attr`, unless that causes some formatting issues?
```suggestion
			<textarea name="srm_redirect_rule_message" id="srm_redirect_rule_message" class="widefat"><?php echo esc_textarea( $redirect_message ); ?></textarea>
```

Reply:
I see the below code is already using `esc_attr` which is why we probably did the same here. I'd suggest testing to see if `esc_textarea` works like we want but if not, we can use `esc_attr` instead

Reply:
@dkotter I just checked the code and found that we sanitize the `textarea` data using `sanitize_text_field` instead of `sanitize_textarea_field` before saving it to DB. It wouldn't make much of a difference using `esc_attr` vs `esc_textarea` in this case.